gothemis/keys/symmetric.go

```diff
@@ -25,6 +25,12 @@ import (
 	"github.com/cossacklabs/themis/gothemis/errors"
 )
 
+// Errors returned by key generation.
+var (
+	ErrGetSymmKeySize   = errors.New("Failed to get symmetric key size")
+	ErrGenSymmKey       = errors.New("Failed to generate symmetric key")
```

Comment:
This can be similar to keypair generation, with full words:

- `ErrGetSymmetricKeySize`
- `ErrGenerateSymmetricKey`